about.html

```diff
@@ -105,17 +105,17 @@ <h2 property="schema:name">Features</h2>
                   <p><strong>Organizations Deploying Solid</strong></p>
                   <p>See organizations working to support the deployment of Solid technology across a range of
                   sectors.</p>
-                  <a href="" rel="rdfs:seeAlso">See organizations</a>
+                  <a href="/for-organizations.html" rel="rdfs:seeAlso">See organizations</a>
                 </li>
                 <li>
                   <p><strong>Develop a Solid application</strong></p>
                   <p>Find tutorials, repositories, libraries, and vocabularies.</p>
-                  <a href="" rel="rdfs:seeAlso">Develop an application</a>
+                  <a href="https://forum.solidproject.org/c/build-a-solid-app" rel="rdfs:seeAlso">Develop an application</a>
```

Comment:
I think @gisellewenban's idea was to link this to the /for-developers.html page, but this looks good too; I'll leave it to her to decide

Reply:
`/for-developers` may be most appropriate if it can refer to various key places for what's intended here. The forum is just one among many.

Reply:
Done!